src/promptflow-devkit/promptflow/_cli/_params.py

```diff
@@ -356,3 +356,13 @@ def add_param_init(parser):
         "Example: --init param1=val1 param2=val2",
         nargs="+",
     )
+
+
+def add_param_path(parser):
+    parser.add_argument(
```

Comment:
please also add changelog and reference doc in this PR

Reply:
done